### DISCUSSION TITLE
Sign a PDF without a certificate

### DISCUSSION THREAD

**Discussion by asada931** [MatthiasValvekens/pyHanko#330] [Q&A] [answered]
I know that PyHanko's main purpose is to digitally sign PDF documents using certificates. However, I have a use case where I just want to display the signature (stored as an image for example in the database) on the PDF document. Can I achieve this using PyHanko or is it must to have a certificate and private key to sign PDFs ? Thanks in advance

**Comment by MatthiasValvekens** [comment, marked as answer]
Well, that's not really what I'd call a PDF signature, but as a matter of technical capabilities in pyHanko: yes, you can do that by setting the image as a stamp background with no content. https://pyhanko.readthedocs.io/en/latest/cli-guide/stamping.html

**Reply by asada931** [reply]
That's exactly what I was looking for. many thanks!
